### ISSUE TITLE
Support Dart 2.1

### ISSUE THREAD

**Issue by TiiRiiX** [kirbyfan64/vuedart#27]
Hi! 15 November [released](https://medium.com/dartlang/announcing-dart-2-1-improved-performance-usability-9f55fca6f31a) Dart 2.1. Is it planned to support it in the near future? 

Now it is not compatible with
```
build_runner: ^1.1.2
build_web_compilers: ^0.4.0
```

**Comment by kirbyfan64** [comment]
Dang it, that was fast :flushed:. I'll try upgrading it later this week!

**Comment by kirbyfan64** [comment]
Hmm, @TiiRiiX it seems to be working for me with Dart 2.1 from the Docker image. Can you post the exact errors pub gave you?

**Comment by TiiRiiX** [comment]
@kirbyfan64 

I have this in console:
```
"C:\Program Files\Dart\dart-sdk\bin\pub.bat" get
Resolving dependencies...
Because no versions of build_runner match >1.1.2 <2.0.0 and build_runner 1.1.2 depends on build >=1.0.0 <1.1.0, build_runner ^1.1.2 requires build >=1.0.0 <1.1.0.
And because vue >=0.4.0 depends on build ^0.12.7, build_runner ^1.1.2 is incompatible with vue >=0.4.0.
So, because vue_test depends on both vue ^0.4.0 and build_runner ^1.1.2, version solving failed.
Process finished with exit code 1
```

with this part of `pubspec.yaml`
```
environment:
  sdk: '>=2.1.0 <3.0.0'

dependencies:
  vue: ^0.4.0

dev_dependencies:
  build_runner: ^1.1.2
  build_web_compilers: ^0.4.0
```

**Comment by kirbyfan64** [comment]
@TiiRiiX Ah, I see the problem...it was my bad. I just released 0.4.1 with the upgraded dependency!
